tools/ci_build/exclude_unused_ops.py

```diff
@@ -24,6 +24,17 @@
               'com.intel.ai': 'kNGraphDomain',
               'com.xilinx': 'kVitisAIDomain'}
 
+# deprecated ops where the last registration should have an end version.
+# value for each entry is the opset when it was deprecated. end version of last registration should equal value - 1.
+deprecated_ops = {'kOnnxDomain:Scatter': 11,
```

Comment:
>deprecated_ops [](start = 0, length = 14)

If the filtered ops has an deprecated op, script will exit without doing the job right?

Reply:
Refactored to separate reduction from validation as the validation requires the original kernel registration file.

---
In reply to: [523937948](https://github.com/microsoft/onnxruntime/pull/5817#discussion_r523937948) [](ancestors = 523937948)